src/Client/ClientApplicationBase.cpp

```diff
@@ -240,8 +241,12 @@ void ClientApplicationBase::init(int argc, char ** argv)
     fatal_channel_ptr = new Poco::SplitterChannel;
     fatal_console_channel_ptr = new Poco::ConsoleChannel;
     fatal_channel_ptr->addChannel(fatal_console_channel_ptr);
+
     if (options.count("client_logs_file"))
     {
+        if (client_context->getApplicationType() == Context::ApplicationType::EMBEDDED_CLIENT)
```

Comment:
We can introduce an option in `ClientOptions` for this as well (yes, even though it is not possible to pass this settings now)